ice.c

```diff
@@ -1245,6 +1245,8 @@ void janus_ice_stream_free(janus_ice_stream *stream) {
 	stream->video_rtcp_ctx[1] = NULL;
 	g_free(stream->video_rtcp_ctx[2]);
 	stream->video_rtcp_ctx[2] = NULL;
+	g_slist_free(stream->transport_wide_received_seq_nums);
```

Comment:
This is a list of allocated elements (pointers to `janus_rtcp_transport_wide_cc_stats` objects), but the allocated values are never freed. You probably want a [g_slist_free_full](https://developer.gnome.org/glib/stable/glib-Singly-Linked-Lists.html#g-slist-free-full) here, that uses `g_free` as the free function.